dcm4che-net-audit/src/main/java/org/dcm4che3/net/audit/AuditLogger.java

```diff
@@ -637,30 +637,29 @@ public void setSupplement95(boolean sup95) {
 
     public boolean isInstalled() {
         return device != null && device.isInstalled()
-                && (auditLoggerInstalled == null || auditLoggerInstalled.booleanValue());
+                && auditLoggerInstalled;
```

Comment:
Won't this break the current behavior? Are there any use cases where AuditLoggerInstalled can be null?

Reply:
I think the auditLoggerInstalled is null only when it's first set to it as the default value. I don't see why it needs to be null otherwise.
The old logic is causing audit messages being generated even if auditing is disabled.

Reply:
No. Should behave like the Installed field of [Network Application Entity](http://dicom.nema.org/medical/dicom/current/output/html/part15.html#sect_H.1.1.2) and [Network Connection](http://dicom.nema.org/medical/dicom/current/output/html/part15.html#sect_H.1.1.3):

> A Boolean value. True if the Audit Logger is installed. If null, information about the installed status of the Audit Logger  is inherited from the device.